xml/en/download.xml

```diff
@@ -35,32 +35,32 @@
 <section name="Source Code">
 
 <para>
-Read-only Mercurial repositories:
+GitHub repositories:
 <list type="bullet">
 
 <listitem>
-code: <literal>http://hg.nginx.org/nginx</literal>
+code: <link url="https://github.com/nginx/nginx"/>
 </listitem>
 
 <listitem>
-website: <literal>http://hg.nginx.org/nginx.org</literal>
-(mirror of GitHub repository)
+website: <link url="https://github.com/nginx/nginx.org"/>
 </listitem>
 
 </list>
 </para>
 
 <para>
-GitHub repositories:
+Read-only Mercurial repositories:
```

Comment:
"Read-only Mercurial repositories" was used to emphasize that these repo cannot be used to pushing changes directly, after repo shift it lost its sense.
Please replace it with "Mercurial mirrors" and remove text in parentheses.